components/logstash/scripts/start.py

```diff
@@ -15,3 +15,8 @@
 ctx.logger.info('Starting Logstash Service...')
 utils.start_service_and_archive_properties(LOGSTASH_SERVICE_NAME,
                                            append_prefix=False)
+
+if utils.systemd.is_alive(LOGSTASH_SERVICE_NAME, append_prefix=False):
```

Comment:
the logging for each `is_alive` check should probably move to the `is_alive` method as it's repeated everywhere.